### PR TITLE
input/locale: split hardware and virtual keyboard setting in strings.po

### PR DESCRIPTION
**Pull request by Portisch**
## Description

Split hardware and virtual keyboard setting in strings.po.
Using the same string for both "Keyboard layouts" for both is just confusing.

## Motivation and Context

Clean keyboard layout settings

## How Has This Been Tested?

On CoreELEC 19 Matrix with Kodi Matrix 19.0 beta2

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [x] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [x] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Comment by CvH** [comment]
any news about this ?
8 months and nobody even looked at it !?

**Comment by Portisch** [comment]
almost same here:
https://github.com/xbmc/xbmc/pull/18975

23.12.2020, tested and used since 04.2019.
Maybe with Kodi 30.

no wonder that the PRs become less every year...

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -1926,7 +1926,11 @@ msgctxt "#406"
 msgid "Humidity"
 msgstr ""
 
-#empty strings from id 407 to 408
+msgctxt "#407"
+msgid "Hardware keyboard layout"
```
Comment (howie-f) [inline addons/resource.language.en_gb/resources/strings.po:1930]: plural here? `Hardware keyboard layouts`

File: addons/resource.language.en_gb/resources/strings.po [hunk 2]
```diff
@@ -20278,10 +20282,10 @@ msgstr ""
 
 #empty strings from id 36435 to 36435
 
-#. Description for setting #310: "Keyboard layouts"
+#. Description for setting #407: "Keyboard layouts"
```
Comment (howie-f) [inline addons/resource.language.en_gb/resources/strings.po:20285]: please also adjust the description: Hardware keyboard layouts

**Review by enen92 — APPROVED** [review]

**Review by howie-f — CHANGES_REQUESTED** [review]
sorry @Portisch, don't want to bother you. an oversight.
i think we need to change the label from old `310` to new `407` in
```
system/settings/freebsd.xml
system/settings/settings.xml (not sure here)
```

**Comment by Portisch** [comment]
> system/settings/freebsd.xml

This is a link to `linux.xml` and there it's already changed as it's the #407 for virtual keyboard.
And #310 for `settings.xml` should be correct as it's for the hardware keyboard description.

**Review by howie-f — APPROVED** [review]
thanks for your contribution

**Pull request merged**